src/autopas/containers/verletListsCellBased/verletLists/VerletLists.h

```diff
@@ -56,20 +58,22 @@ class VerletLists
    * The neighbor lists are build using a search radius of cutoff + skin.
    * The rebuildFrequency should be chosen, s.t. the particles do not move more
    * than a distance of skin/2 between two rebuilds of the lists.
-   * @param boxMin the lower corner of the domain
-   * @param boxMax the upper corner of the domain
-   * @param cutoff the cutoff radius of the interaction
-   * @param skin the skin radius
-   * @param rebuildFrequency specifies after how many pair-wise traversals the
+   * @param boxMin The lower corner of the domain
```

Comment:
```suggestion
   * @param boxMin The lower corner of the domain.
```
Also applies to the following lines :P